frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardCountBlock.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+
+const MonitoringDasboardCountBlock = () => {
```

Comment:
nit: we have type `React.FC` and do implicit  return

Reply:
Done

Reply:
FWIW @invincibleJai this is more than just a nit, it's a requirement. We don't want to create anything without types if we can avoid it.